src/main/kotlin/org/openmbee/flexo/mms/routes/gsp/ModelLoad.kt

```diff
@@ -516,12 +516,39 @@ suspend fun GspLayer1Context<GspMutateResponse>.loadModel() {
         )
     }
 
+    val constructCommitString = buildSparqlQuery {
+        construct {
+            txn("load")
```

Comment:
i don't think we need the load txn results here. in fact, we might as well just query for the commit triples here and concatenate with `diffConstructResponseText`

Reply:
yea i was thinking that - there's gonna be more simplifying in the mutex branch, so I think i'm gonna merge to that branch for the tests and fix things there as part of the overhaul